src/components/results/candidate-results/personality-archetype/index.js

```diff
@@ -33,24 +34,50 @@ class PersonalityArchetype extends Component {
     const personality = this.props.assessment.archetype;
     if(!personality) { return null; }
     const badge = personality.details.find(({title}) => title === "Badge");
-    const description = personality.details.find(({title}) => title === "Candidate Description");
+    const perspective = this.props.getOption("perspective");
+    const description = (
+      perspective === "thirdPerson"
+        ? personality.details.find(({title}) => title === "Hiring Manager Description")
+        : personality.details.find(({title}) => title === "Candidate Description")
+    );
 
     return (
       <div className={style.container}>
-        <div className={style.details}>
-          {badge && <img alt={personality.name} src={badge.body} />}
-          <DangerousHTML
-            html={this.props.translate("personality_heading", {
-              deck_name: this.props.deck.name,
-              personality: `<span>${personality.name}</span>`
-            })}
-            tag="h2"
-          />
-          {description && <p>{description.body}</p>}
-        </div>
-        <div className={style.meaning}>
-          <DangerousHTML html={this.props.translate("candidate_description_for_archetype_html")} tag="p" />
-        </div>
+        {perspective === "thirdPerson" ? (
+          <div className={style.thirdPersonDetails}>
+            <div className={style.itemOne}>
```

Comment:
For some reason description is missing for me (maybe just not in staging right now). But when description is gone, I think we need to remove this class or update it to not have the bar

Reply:
I think if you change where the api points to to staging it should work. I did make the update for the bar to be conditional